src/main/java/Task.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
```

Comment:
The coding standards discourage importing an entire package. It is easier to read the specific classes the code uses, such as the ones included below (List and Stream)